cli/specifier_handler.rs

```diff
@@ -25,8 +26,30 @@ pub type DependencyMap = HashMap<String, Dependency>;
 pub type FetchFuture =
   Pin<Box<(dyn Future<Output = Result<CachedModule, AnyError>> + 'static)>>;
 
+/// A group of errors that represent errors that can occur with an
+/// an implementation of `SpecifierHandler`.
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub enum HandlerError {
+  /// A fetch error, where we have a location associated with it.
+  FetchErrorWithLocation(String, Location),
+}
+use HandlerError::*;
```

Comment:
FYI I'm not a fan of doing that - it pollutes whole module with all the variants - if it's only required for `fmt::Display` I suggest moving it into `fn fmt()`

Reply:
It is only for convenience...  I will remove it.